Add missing KIC 3.0 docs (#6530)
Kong/docs.konghq.com: app/_src/kubernetes-ingress-controller/guides/customize/image.md

---
title: Customize Images
---

{{ site.kic_product_name }} supports both {{ site.base_gateway }} OSS and Enterprise. If you install {{ site.kic_product_name }} using the default Helm charts, {{ site.base_gateway }} OSS will be installed.

You can customize the products used by setting `gateway.image` and `controller.image` in your `values.yaml`. Once this is done, upgrade your Helm deployment.

```bash
helm upgrade <release_name> kong/ingress -n <namespace> --values ./values.yaml
```

You can check which images are being used with the following query:

```bash
kubectl get pods -n kong -o jsonpath="{.items[*].spec.containers[*].image}"
```

## Common Images

> The following `values.yaml` files are commonly used by customers.

Reviewer: ```suggestion
These are some of the commonly used `values.yaml` files.
```
